utils/helper.py

```diff
@@ -184,4 +200,34 @@ def credits_to_string(amount: int) -> str:
     if amount >= 10**18:
         return '{:,} {}C'.format(int(amount / divider), letter)
     else:
-        return '{:.3g} {}C'.format(amount / divider, letter)
+        power_of_10 = max(0,int(math.floor(math.log10(absAmount))))
+        precision = significant_numbers - 1 - (power_of_10 % 3)
+        return '{1:.{0}f} {2}C'.format(precision,
+            math.floor(amount / 10**(power_of_10 - significant_numbers + 1)) / 10**precision, 
+            letter)
+
+def credits_to_string_with_exact_value(amount: int, separator: str = ' ', significant_numbers: int = 3) -> str:
+    """Returns "XXX'XXX" C if under a million, otherwise "XXX MC (XXX'XXX'XXX C)" """
+    if amount >= 10**6:
+        return '{}{}({:,} C)'.format(credits_to_string(amount), separator, amount)
+    else:
+        return '{:,} C'.format(amount)
+
+async def bot_log(client: discord.client, error: str, message: discord.Message = None):
+    # Print the error in the console, then in #bot-log channel if it exists
+    error_message = '{}{}'.format(
+            '{} in #{}: "{}": '.format(
+                message.author.display_name, 
+                message.channel,
+                message.content) if message is not None else '', 
+            error);
+
+    # Console
+    print(error_message)
+
+    # Discord channel
+    channel_name = 'bot-log'
```

Comment:
There already exists a channel, it's called bot-logs, please rename this to plural